Menu/menu.asm

```diff
@@ -158,11 +160,10 @@ Menu_ScrollDown:
   INX : INX
   LDA.w Menu_Scroll, X 
   STA.b $EA
-  CMP.w #$FF12 : BNE .loop
-
-  JMP Menu_InitItemScreen
+  CMP.w #$FF12 : BNE .notDoneScrolling
+    JMP Menu_InitItemScreen
 
-.loop
+  .notDoneScrolling
   STX.w MenuScrollLevelV

```

Comment:
Let's indent the code beneath the labels like the loop earlier in this file

Reply:
what? i'm confused, its not actually a loop, the label was wrong.

Reply:
Oh I misread. This should be fine then.